config/environments/production.rb

```diff
@@ -68,12 +68,12 @@
   config.assets.digest = true
 
   # Defaults to Rails.root.join("public/assets")
-  config.assets.manifest = Rails.root.join('public/assets')
+  # config.assets.manifest = Rails.root.join('public/assets')
```

Comment:
It was throwing an error since `sprockets` was trying to open the file `public/assets` but it's actually a directory (see [this comment](https://github.com/YaleSTC/reservations/issues/275#issuecomment-69699081)). Assets generally appear to be working (at least CSS / JS), I'm not sure about images from the seed script.